test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -44,20 +46,31 @@ using ::testing::_;
 class YacGateTest : public ::testing::Test {
  public:
   void SetUp() override {
+    auto keypair =
+        shared_model::crypto::DefaultCryptoAlgorithmType::generateKeypair();
+
     expected_hash = YacHash("proposal", "block");
-    old_expected_block.sigs.emplace_back();
-    old_expected_block.sigs.back().pubkey.fill(1);
-    expected_hash.block_signature = old_expected_block.sigs.front();
+    shared_model::proto::Block tmp =
+        shared_model::proto::BlockBuilder()
+            .height(1)
+            .createdTime(iroha::time::now())
+            .transactions(std::vector<shared_model::proto::Transaction>{})
+            .prevHash(Sha3_256::makeHash(Blob("block")))
+            .build()
+            .signAndAddSignature(keypair);
+
+    expected_block =
+        std::unique_ptr<shared_model::interface::Block>(tmp.copy());
+    const auto &signature = **expected_block->signatures().begin();
```

Comment:
Maybe add brackets for `*` operator:
```
 *(*expected_block)->signatures().begin();
```

Reply:
Why? The order of this operator is determined